### DISCUSSION TITLE
how to run raspi cam srv

### DISCUSSION THREAD

**Discussion by hardikjp7** [signag/raspi-cam-srv#10] [Q&A] [answered]
how to run raspi cam srv

**Comment by signag** [comment, marked as answer]
After you have installed raspiCamSrv following the [Installation Instructions](https://github.com/signag/raspi-cam-srv?tab=readme-ov-file#raspicamsrv-installation), you have two alternatives:

1. Start the server from the command line as described in step 13
2. Set up and start a service as described in [Service Configuration](https://github.com/signag/raspi-cam-srv?tab=readme-ov-file#service-configuration)

**Reply by hardikjp7** [reply]
I apologize, I made a mistake by not reading all the instructions first and just went straight into the folder.  Thankfully, it all worked out in the end!  I can't thank you enough, this repository saved me weeks of work on getting direct camera input.
